File: Selenium/Activities/Activity22.java
package example;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity22 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		WebDriverWait wait = new WebDriverWait(driver,Duration.ofSeconds(10));

		driver.get("https://training-support.net/webelements/popups");
		System.out.println("Parent Title of the page is: " + driver.getTitle());
		driver.findElement(By.id("launcher")).click();
		wait.until(ExpectedConditions.elementToBeClickable(By.id("username")));
		
		
		driver.findElement(By.id("username")).sendKeys("admin");
		driver.findElement(By.id("password")).sendKeys("password");
		driver.findElement(By.xpath("//button[text()='Submit']")).click();
		System.out.println(driver.findElement(By.cssSelector("h1.text-center")).getText());
		driver.quit();

	}

}

File: Selenium/Activities/Activity22.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as fireFoxService
from selenium.webdriver.common.by import By
import time
from selenium.webdriver.common.action_chains import ActionChains


service=fireFoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service)as driver:
    driver.get("https://www.training-support.net/selenium/popups")
    print("Page Title :- " + driver.title)
    ele=driver.find_element(By.CLASS_NAME,"orange")
    act=ActionChains(driver)
    act.move_to_element(ele).perform()
    toolTeap=ele.get_attribute("data-tooltip")
    print("Tool Teap:- " + toolTeap)
    ele.click()
    driver.find_element(By.ID,"username").send_keys("admin")
    driver.find_element(By.ID,"password").send_keys("password")
    driver.find_element(By.XPATH,"//button[text()='Log in']").click()
    message=driver.find_element(By.ID,"action-confirmation").text
    print("Confirmation Message :- " + message)
    driver.close()


    
